ansible/container-infra-configure.yml

```diff
@@ -11,10 +11,8 @@
       os_manila_mount_share_name: "{{ ceph_mount_share_name }}"
       os_manila_mount_share_user: "home"
 
-- hosts: container-infra
+- hosts: cluster
   gather_facts: yes
-  vars:
-    ib_ip: "{{ hostvars[inventory_hostname]['server_networks'][lln_name][0] }}"
```

Comment:
Will have to find a way of adapting this to container-infra in the next iteration.

Reply:
Perhaps by making config_drive an optional source of info?

Reply:
If you separate the discovery of the IP address from the configuration then you can share the config. i.e. pass ib_ipaddr in as a variable to the role, and determine its value externally, in the a play tasks or another role.

All that said, if the cloud-init & config-drive config worked (which it now should, right?) we don't need to read the config drive do we?